services/static-webserver/client/source/class/osparc/product/Utils.js

```diff
@@ -79,6 +95,55 @@ qx.Class.define("osparc.product.Utils", {
           break;
       }
       return logosPath;
+    },
+
+    showLicenseExtra: function() {
+      if (this.isProduct("s4llite") || this.isProduct("tis")) {
```

Comment:
THOUGHT: would't it make sense to make this part of the product properties in the database? I will add the ``dev`` column to have a temporary placeholder to some data before definiting it's latest state

Reply:
Not for now 👍